### PR TITLE
add resource limit flags

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: templates/docker-compose.yml.j2 [hunk 1]
```diff
@@ -96,3 +96,5 @@ services:
 {% for node in go_waku_static_nodes %}
       --staticnode={{ node }}
 {% endfor %}
+--resource-scaling-memory-percentage={{ go_waku_resource_scaling_memory_percentage | float }}
+--resource-scaling-fd-percentage={{ go_waku_resource_scaling_fd_percentage | float }}
```
Comment (jakubgs) [inline templates/docker-compose.yml.j2]: The indent is wrong.

File: defaults/main.yml [hunk 2]
```diff
@@ -69,6 +69,10 @@ go_waku_mem_limit: '{{ (ansible_memtotal_mb * 0.6) | int }}'
 go_waku_cpu_reserve: 0.20
 go_waku_mem_reserve: '{{ (ansible_memtotal_mb * 0.2) | int }}'
 
+# Resource Scaling
+go_waku_resource_scaling_memory_percentage: 50
+go_waku_resource_scaling_fd_percentage: 50
```
Comment (jakubgs) [inline defaults/main.yml:74]: Is `50` the default in the code?

Comment (richard-ramos) [reply defaults/main.yml:74]: the default for memory is 25. Will update this

File: templates/docker-compose.yml.j2 [hunk 3]
```diff
@@ -96,3 +96,5 @@ services:
 {% for node in go_waku_static_nodes %}
       --staticnode={{ node }}
 {% endfor %}
+      --resource-scaling-memory-percentage={{ go_waku_resource_scaling_memory_percentage | float }}
+      --resource-scaling-fd-percentage={{ go_waku_resource_scaling_fd_percentage | float }}
```
Comment (jakubgs) [inline templates/docker-compose.yml.j2:100]: Wrong name of flag, fixed in: https://github.com/waku-org/go-waku/commit/0dbe4fd5ff4f647f455b3af08c02910c4e0d8011

**Review by jakubgs — APPROVED** [review]

**Pull request merged**